src/eShop.AppHost/Program.cs

```diff
@@ -68,16 +72,16 @@
     .WithReference(basketApi)
     .WithReference(catalogApi)
     .WithReference(orderingApi)
-    .WithReference(rabbitMq)
+    .WithReference(rabbitMq).WaitFor(rabbitMq)
     .WithEnvironment("IdentityUrl", identityEndpoint);
 
 // set to true if you want to use OpenAI
-bool useOpenAI = false;
+bool useOpenAI = true;
```

Comment:
```suggestion
bool useOpenAI = false;
```

We should probably keep this off by default.